### DISCUSSION TITLE
Welcome to yobulkdev Discussions!

### DISCUSSION THREAD

**Discussion by YoBulkMaster** [yobulkdev/yobulkdev#2] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AlessandroSalvetti** [comment]
Hi,
first of all thanks so much about the project, it's amazing!
I have a question about mapping: once the user have done the mapping between CSV and template, is there a way to save this mapping? So that the second time the user loads the CSV doesn't need to set the mapping again

**Comment by YoBulkMaster** [comment, marked as answer]
@AlessandroSalvetti No Right now, we are not saving the mapping.I am just adding it to our roadmap.Thanks for suggestion and query.

**Reply by AlessandroSalvetti** [reply]
Okay, thank you very much

**Comment by drish** [comment]
Hey, first of all thanks for building this project !

I wanted to ask you a question about SaaS integration. From what I could see so far there is no concept of JobID or UploadID, so after the user uploads a file, how could we programmatically fetch that specific file back from the API ?
